INSTALL.md

```diff
@@ -310,7 +310,7 @@ together one-size-fits-all instructions.  You might have to pass more flags or
 set up environment variables to actually make it work.  Android and iOS cases
 are discussed in corresponding `Configurations/15-*.conf` files.  But there are
 cases when this option alone is sufficient.  For example to build the mingw64
-target on Linux `--cross-compile-prefix=x86_64-w64-mingw32-` works.  Naturally
+target on Linux `--cross-compile-prefix=x86_64-w64-mingw32-` works.  Naturally,
```

Comment:
If might be better to just remove `Naturally`:

`For example to build the mingw64 target on Linux '--cross-compile-prefix=x86_64-w64-mingw32- works (provided that mingw packages are installed)`